src/pfx_ng_fmt_plug.c

```diff
@@ -118,7 +119,7 @@ static int valid(char *ciphertext, struct fmt_main *self)
 	if (!isdec(p))
 		goto bail;
 	mac_algo = atoi(p);
-	if (mac_algo != 1) // 1 -> SHA1
+	if (mac_algo != 1 && mac_algo != 256) // 1 -> SHA1, 256 -> SHA256
 		goto bail;
```

Comment:
I will add this code, now that we have this merged, and get my SIMD code in there, so that when sha512 is added, we are ready for SIMD code for it (that may take a bit more work).